src/main/java/duke/parser/Parser.java

```diff
@@ -1,6 +1,12 @@
 package duke.parser;
 
-import duke.commands.*;
```

Comment:
Good job omitting the wildcard import and replacing it with very specific imports! :)